### DISCUSSION TITLE
chess engine analysis more human readable

### DISCUSSION THREAD

**Discussion by sugizo** [niklasf/python-chess#994] [Ideas]
**generated by python-chess module**
![image](https://user-images.githubusercontent.com/2418295/233850879-71a02c7d-8635-4521-ad98-37e6393693aa.png)

**generated by stockfish python module**
![image](https://user-images.githubusercontent.com/2418295/233850919-a22cd7b5-ffb8-47c9-afe1-24d5419d2494.png)

nice to make the chess engine analysis more human readable like stockfish python module

also, not sure about : 
cp = centipawn ??

povscore (is it _point of view_ ? ) 
and 
pv (is it pricinipal variation or predicted variation)

**ref** 
https://www.chessprogramming.org/Principal_Variation
https://www.chessprogramming.org/Extended_Position_Description#Opcode_mnemonics

thanks and best regards

**Comment by niklasf** [comment]
`PovScore(Cp(+18), WHITE)` is a debug representation for developers, showing exactly the composition of the given object. The purpose of [`PovScore`](https://python-chess.readthedocs.io/en/latest/engine.html#chess.engine.PovScore) is indeed to encourage correctly handling the point of view. When trying to access the inner [`Score`](https://python-chess.readthedocs.io/en/latest/engine.html#chess.engine.Score), a concusious choice about the point of view has to be made.

For example, you could use `pov_score.white()` to always take White's point of view.

```python
>>> import chess.engine

>>> pov_score = chess.engine.PovScore(chess.engine.Cp(+18), chess.WHITE)
>>> pov_score
PovScore(Cp(+18), WHITE)
>>> pov_score.white()
Cp(+18)
>>> str(pov_score.white())  # Get a nice string representation for humans
'+18'

>>> pov_score = chess.engine.PovScore(chess.engine.Mate(3), chess.BLACK)
>>> pov_score
PovScore(Mate(+3), BLACK)
>>> pov_score.white()
Mate(-3)
>>> str(pov_score.white())
'#-3'
```
